### ISSUE TITLE
Duplicated List Markup Generated

### ISSUE THREAD

**Issue by cjames9001** [Sicos1977/MSGReader#117]
As a test I've sent myself an email from my GoDaddy email account using the online rtf editor for emails. The bug I'm reporting is that lists are double generating letters and bullets.

I can provide the actual .msg file if needed or anything else that may help.

From outlook:
![renderedinoutlook](https://user-images.githubusercontent.com/11615850/43281817-410dd658-90da-11e8-9b92-d710f48cf1c3.PNG)

From MSGViewer:
![renderedviamsgviewer](https://user-images.githubusercontent.com/11615850/43281996-a98f4540-90da-11e8-851a-e0b6bec15bf7.PNG)

Side note, besides this little bug, this project is great. Thank you so much for this.

**Comment by Sicos1977** [comment]
Your welcome and can you please send me the msg file? You can sent it to sicos2002@hotmail.com

**Comment by cjames9001** [comment]
I just re-fowarded the email to you. The subject is “Duplicated List Markup Generated” in case it ended up in your spam. Thanks!

**Comment by Sicos1977** [comment]
It ended up in my spam folder again but now I got the msg file. I'll try to look into it next evening.

**Comment by Sicos1977** [comment]
And thanks for the nice words about my MSGReader

**Comment by Sicos1977** [comment]
Can you please check if this comit 13a870e4454a7e10399f0755f412c705677df5c8 fixes your issue. It does over here. Please reopen the issue if it isn't.

![image](https://user-images.githubusercontent.com/6692947/43667271-820c1a5e-9777-11e8-93da-08a721bbdcc8.png)

**Comment by Sicos1977** [comment]
P.S. just for my own curiosity... for what are you using the MSGReader?

**Comment by cjames9001** [comment]
Worked like a charm. Thanks! 

I have a project that generates document previews. Part of that process uses Outlook, which causes some headaches & doesn't generate the previews nearly as nice as your project does. Outlook can do plain text previews pretty well, but as soon as there's any advanced formatting it falls flat.

**Comment by Sicos1977** [comment]
> Outlook can do plain text previews pretty well, but as soon as there's any advanced formatting it falls flat.

Outlook also isn't thread safe. I used to do something similar. I needed Outlook to convert MSG to HTML to PDF format (with wkHtmlToPdf ... what I now do with my other project ChromeHtmlToPdf). 

But I got all kind of strange errors when using the code on a server. So I started developing MSGReader to overcome these issues. A bonus was also that MSGReader is lightning fast compared to spinning up Outlook every time.
